### PR TITLE
Pre release v0.4.2

### LINKED ISSUE #203: New release v0.4.2
Plan to release new version v0.4.2, with the new contents:
- Support of LLVM 11
- Option to add abolute or relative error
- Option to read VFC_BACKENDS from file

### PR DESCRIPTION
**Pull request by yohanchatelain**
# Added
  * Support for LLVM up to 11.0.1
  * Mechanism for reading VFC_BACKENDS from file through VFC_BACKENDS_FROM_FILE
  * Absolute error option for MCA and VPREC backends

Closes #203

### CODE REVIEW

**Review by pablooliveira — APPROVED** [review]
LGTM :-) Thanks !

File: Dockerfile [hunk 1]
```diff
@@ -4,7 +4,7 @@
 #
 
 FROM ubuntu:20.04
-MAINTAINER Verificarlo contributors <verificarlo@googlegroups.com>
+LABEL maintainner="verificarlo contributors <verificarlo@googlegroups.com>"
```
Comment (pablooliveira) [inline Dockerfile:7]: Ooops. maintainner -> maintainer.

**Review by pablooliveira — APPROVED** [review]
Sorry just saw a typo in the Dockerfile LABEL. Still approving since it's a super minor change.

File: Dockerfile (hunk 1 above)
Comment (yohanchatelain) [reply Dockerfile:7]: Good point!

**Pull request merged**